### PR TITLE
Adding notes to document Bug 1341312 and Bug 1319906

### PR DESCRIPTION
**Pull request by sferich888**
Supplying docs for https://bugzilla.redhat.com/show_bug.cgi?id=1319906 and https://bugzilla.redhat.com/show_bug.cgi?id=1341312

### CODE REVIEW

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -147,6 +147,28 @@ addresses; because of the NAT configuration, the originating IP address
 `*hostNetwork: true*`, all external clients will be routed to a single pod.
 ====
 
+*Configuration Paramiters*
```
Comment (ahardin-rh) [inline architecture/core_concepts/routes.adoc]: s/Paramiters/Parameters

File: architecture/core_concepts/routes.adoc [hunk 2]
```diff
@@ -147,6 +147,28 @@ addresses; because of the NAT configuration, the originating IP address
...
+.Router Configuration Paramiters
+[cols="3*", options="header"]
+|===
+|Variable | Default
+|HAPROXY_ROUTER_SYSLOG_ADDRESS | "" 
+|HAPROXY_ROUTER_LOG_LEVEL | warning
+|ROUTER_BACKEN_CHECK_INTER | 5000ms
+|===
+
+As an example you can apply the paramters listed using the example:
```
Comment (ahardin-rh) [inline architecture/core_concepts/routes.adoc]: Comma after example

s/paramters/parameters

File: architecture/core_concepts/routes.adoc [hunk 3]
```diff
@@ -147,6 +147,28 @@ addresses; because of the NAT configuration, the originating IP address
 `*hostNetwork: true*`, all external clients will be routed to a single pod.
 ====
 
+*Configuration Paramiters*
+
+With all the items outlined in this section you should be able to set enviroment
```
Comment (ahardin-rh) [inline architecture/core_concepts/routes.adoc]: comma after section

**Comment by ahardin-rh** [comment]
@sferich888 Thanks! Just a few comments from me. Also, please check that my labels and milestone are correct.

**Comment by sferich888** [comment]
I have made the fixes, for this as requested, however the merge needs to wait until https://github.com/openshift/origin/pull/9099 is accepted.

**Comment by ahardin-rh** [comment]
@sferich888 Is this ready to go? Looks like it just needs a rebase. Thanks!

**Comment by ahardin-rh** [comment]
Work completed in https://github.com/openshift/openshift-docs/pull/2183

**Pull request closed without merging**